cmd/kpod/kill.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"os"
+)
+
+var (
+	killFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "signal, s",
+			Usage: "Signal to send to the container",
+			Value: "KILL",
+		},
+	}
+	killDescription = "Kill one or more running containers"
```

Comment:
Why not 
killDescription = `
       The main process inside each container specified will be sent SIGKILL,
        or any signal specified with option --signal.
`

Reply:
fixed